kv/main.go

```diff
@@ -585,6 +587,7 @@ func main() {
 				}
 			}
 
+			cumLatency.Merge(h)
```

Comment:
It returns the number of samples that could not be merged. I believe that can only happen if you merge 2 histograms that were configured differently. Our histograms are all configured identically.